compiler/parser/RdrHsSyn.hs

```diff
@@ -3102,8 +3102,11 @@ mkInterfaceDecl tycon decls = do
             ]
         existentialExerciseInstances :: [LHsDecl GhcPs]
         existentialExerciseInstances =
```

Comment:
We really shouldn't call these existentials 😢

Reply:
yeah it’s not great, I’m not sure "interfaceObjects" is better though (given the context of Daml) and I don’t have a better suggestion.